package.xml

```diff
@@ -2,10 +2,11 @@
 <package format="2">
   <name>prophesee_ros_stereo_driver</name>
   <version>0.0.2</version>
-  <description>Publisher and Viewer for Prophesee Event-Based Stereo Sensors</description>
+  <description>Stereo Publisher and Viewer for Prophesee Event-Based Sensors</description>
 
   <maintainer email="nlyubova@prophesee.ai">Natalia Lyubova</maintainer>
   <author email="natalia.lyubova@prophesee.ai">Natalia Lyubova</author>
+  <author email="egor.makhov@mail.polimi.it">Egor Makhov</author>
```

Comment:
could you please remove this line and add instead of it your name ? as you did the code

Reply:
Done.